### PR TITLE
Added functionality to save the statistics table if the user wants to

### PR DESCRIPTION
**Pull request by voremargot**

### CODE REVIEW

**Review by BooleanJulien — APPROVED** [review]
lgtm!

File: src/strappy/display.py [hunk 1]
```diff
@@ -102,9 +104,10 @@ def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
     
 
 
-def tabulate_stats(stat, precision=2, estimator=True, alpha=True):
+def tabulate_stats(stat, precision=2, estimator=True, alpha=True,  save = False, path= None):
```
Comment (gauthampughazhendhi) [inline src/strappy/display.py]:
```suggestion
def tabulate_stats(stat, precision=2, estimator=True, alpha=True,  path=None):
```

File: src/strappy/display.py [hunk 2]
```diff
@@ -150,6 +157,15 @@ def tabulate_stats(stat, precision=2, estimator=True, alpha=True):
         raise TypeError(
             "The estimator and alpha parameters must be of type boolean."
         )
+    if not (isinstance(path, str) or path is None):
+        raise TypeError("The path parameter must be a character string.")
+        
+    if not isinstance(save, bool):
+        raise TypeError("The save parameter must be of type boolean.")  
+        
+    if path != None :
```
Comment (gauthampughazhendhi) [inline src/strappy/display.py]:
```suggestion
    if path is not None :
```

File: src/strappy/display.py [hunk 3]
```diff
@@ -197,7 +213,13 @@ def tabulate_stats(stat, precision=2, estimator=True, alpha=True):
...
           "props": "caption-side: bottom; font-size: 1.00em;"}],
         overwrite=False)
-
+    
+    if (save == True or path != None):
+        if path != None :
+            stats_table.to_html(f"{path}sampling_statistics.html")
+        else :
+            stats_table.to_html("sampling_statistics.html")
+
```
Comment (gauthampughazhendhi) [inline src/strappy/display.py]:
```suggestion
    if path is not None :
        stats_table.to_html(f"{path}sampling_statistics.html")
    else :
        stats_table.to_html("sampling_statistics.html")
            
```

File: src/strappy/display.py [hunk 4]
```diff
@@ -221,4 +243,10 @@ def tabulate_stats(stat, precision=2, estimator=True, alpha=True):
         overwrite=False)
     )
 
+    if (save == True or path != None):
+        if path != None:
+            bs_params.to_html(f"{path}bootstrap_params.html") 
+        else:
+             bs_params.to_html("bootstrap_params.html") 
+
```
Comment (gauthampughazhendhi) [inline src/strappy/display.py]:
```suggestion
    if path is not None:
        bs_params.to_html(f"{path}bootstrap_params.html") 
    else:
         bs_params.to_html("bootstrap_params.html") 
        
```

**Comment by voremargot** [comment]
Code set up to save html automatically!!

**Pull request merged**